### PR TITLE
Small cleans in beets.util module

### PR DESCRIPTION
**Pull request by brunal**
Looking at the barriers to python 3 I entered `beets/util/__init__.py`. Reading its source took my attention to 2 small functions that could be cleaner and faster. I rewrote them, here it is.

- `str2bool`:  if/else was unwarranted. Note than in python3 using a set will be better (optimized as a frozenset loaded in 1 operation)
- `plurality`: this is what `Counter` is for!

### CODE REVIEW

File: beets/util/__init__.py [hunk 1]
```diff
@@ -544,10 +544,7 @@ def truncate_path(path, length=MAX_FILENAME_LENGTH):
 
 def str2bool(value):
     """Returns a boolean reflecting a human-entered string."""
-    if value.lower() in ('yes', '1', 'true', 't', 'y'):
-        return True
-    else:
-        return False
+    return value.lower() in ('yes', '1', 'true', 't', 'y')
```
Comment (sampsyo) [inline beets/util/__init__.py]: Wow, do I feel dumb for writing that. :flushed:

**Comment by sampsyo** [comment]
These are both awesome. Sadly, I don't think `Counter` is in Python 2.6. And, perplexingly, the tests are failing for a seemingly different reason on 2.6 (something internal to unittest??). The evidence is mounting that we should just ditch 2.6 and be done with it.

**Comment by brunal** [comment]
I pushed the `str2bool` one, while keeping the `plurality` in a `must-drop-py26` branch for later.

**Comment by sampsyo** [comment]
Perfect.

**Pull request closed without merging**